src/vs/workbench/contrib/codeEditor/browser/quickaccess/gotoSymbolQuickAccess.ts

```diff
@@ -194,12 +194,13 @@ export class GotoSymbolQuickAccessProvider extends AbstractGotoSymbolQuickAccess
 						item.highlights = undefined;
 						return true;
 					}
-					const score = fuzzyScore(picker.value, picker.value.toLowerCase(), 1 /*@-character*/, item.label, item.label.toLowerCase(), 0, { firstMatchCanBeWeak: true, boostFullMatch: true });
-					if (!score) {
+					const query = prepareQuery(picker.value.substring(AbstractGotoSymbolQuickAccessProvider.PREFIX.length).trim());
+					const [symbolScore, symbolMatches] = scoreFuzzy2(item.label, query, 0, item.filterStartIndex);
```

Comment:
This changes the way how filtering/ranking works and it is not clear to me why that is needed